packages/run-protocol/test/vaultFactory/test-liquidator.js

```diff
@@ -364,55 +418,14 @@ const makeDriver = async (t, initialPrice, priceBase) => {
         timer.tick();
       }
     },
-    makeVault: async (collateral, debt) => {
-      vaultSeat = await E(zoe).offer(
-        await E(lender).makeVaultInvitation(),
-        harden({
-          give: { Collateral: collateral },
-          want: { RUN: debt },
-        }),
-        harden({
-          Collateral: aethMint.mintPayment(collateral),
-        }),
-      );
-      const {
-        vault: v,
-        publicNotifiers: { vault: vaultNotifier },
-      } = await E(vaultSeat).getOfferResult();
-      t.truthy(await E(vaultSeat).hasExited());
-      vault = v;
-      notifier = vaultNotifier;
-      return vault;
-    },
-    close: async () => {
-      lastSeat = await E(zoe).offer(E(vault).makeCloseInvitation());
-      lastOfferResult = await E(lastSeat).getOfferResult();
-      t.is(lastOfferResult, 'your loan is closed, thank you for your business');
-      t.truthy(await E(vaultSeat).hasExited());
-    },
-    checkNotify: async (phase, expected) => {
-      notification = await E(notifier).getUpdateSince();
-      trace(t, 'notify', notification);
-      t.is(notification.value.vaultState, phase);
-      expected && t.like(notification.value, expected);
-    },
-    awaitNotify: async (phase, expected) => {
-      notification = await E(notifier).getUpdateSince(notification.updateCount);
-      trace(t, 'notify', notification);
-      t.is(notification.value.vaultState, phase);
-      expected && t.like(notification.value, expected);
-    },
+    makeVault: makeVaultDriver,
```

Comment:
`makeVaultDriver` is more accurate. let's keep that name throughout.

otherwise reading below,
```
const v = await d.makeVault(collateralAmount, loanAmount);
```
looks like `v` is a vault made by the driver whereas it's another driver spawned by the main one to wrap a new vault.

consider also keeping `d` in driver objects. that would help in reading tests in code review where most lines (including the assignment) are hidden
```
const vd = await d.makeVaultDriver(collateralAmount, loanAmount);
```
or perhaps a pathname style for when there's more than one driver of vaults:
```
dv1.checkNotify();
dv2.close();
```